agent/logger/log.go

```diff
@@ -86,7 +87,7 @@ func SetLevel(logLevel string) {
 // GetLevel gets the log level
 func GetLevel() string {
 	levelLock.RLock()
-	defer levelLock.RLock()
+	defer levelLock.RUnlock()
```

Comment:
Nit: RUnlock -> Runlock or RunLock??

Reply:
This look like it's a https://godoc.org/sync#RWMutex, so RUnlock is a reader taking a lock.

Reply:
cool

Reply:
RUnlock is unlocking the read-lock.  This is unrelated to Windows; I just saw gometalinter complain about it while I was working in the package.